docker/samtools/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:zesty
+
+LABEL maintainer="Ambrose J. Carr <acarr@broadinstitute.org>" \
+  software="samtools" \
+  version="1.3.1" \
+  description="processing sequence alignments in SAM and BAM formats" \
+  website="https://samtools.github.io"
+
+RUN apt update && apt install -y \
+  samtools
```

Comment:
Sorry I'm a little bit confused here, how do you make sure you are installing the version 1.3.1 of samtools? I saw there the latest version is 1.6 now: (https://github.com/samtools/samtools/tags).

Reply:
Good catch; I'm not sure, actually -- but 1.3.1 is the version that gets installed by apt. I will specify the version explicitly.